tests/FSharp.MongoDB.Bson.Tests/app.config

```diff
@@ -4,7 +4,7 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.3.0.0" newVersion="4.3.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-999.999.999.999" newVersion="4.4.0.0" />
```

Comment:
I have copies this from [here](https://github.com/fsprojects/Paket/blob/master/src/Paket/App.config) to use the most actual version of FSharp.Core available. Otherwise Unquote was thrown missing method exception on my environment. There should be also `nuget FSharp.Core` entry in paket.dependencies to let paket rule here, if I'm not mistaking